plugin/DataView/DataDetailView.php

```diff
@@ -105,6 +105,9 @@ class DataDetailView
 	/** @var string[] */
 	private $_pictureTypes = [];
 
+	/** @var bool */
+	private $_accessControls = true;
```

Comment:
I think some migration code is necessary for this. `$_accessControls` would be `null` in existing views